src/adapters/postgres/user-adapter.ts

```
@@ -524,13 +545,25 @@ export class PostgresUserService implements IServicelocator {
           if (key === 'firstName') {
             whereCondition += ` U."${key}" ILIKE '%${value}%'`;
           } else {
-            if (key === 'status' || key === 'email' || key === 'userId' || key === 'country') {
+            if (
+              key === 'status' ||
```

ESLint

[error] Replace `'status'` with `"status"`

(prettier/prettier)